src/host/main.js

```diff
@@ -68,6 +68,22 @@ export async function hostMain({
 
     const callbackRequests = [];
 
+    const postCallback = body => {
+      // ignore if not in callback mode
+      if (!callbackUrl) return;
+      const headers = {
+        'Content-Type': 'application/json',
+        ...(callbackHeader || {}),
+      };
+      callbackRequests.push(
+        fetch(callbackUrl, {
```

Comment:
This code appears to use the same URL for test status updates as it has already been using for AT responses ("results"). Is that right?

Reply:
I wasn't sure what we were going to do for the API on the frontend about it.  It would be most convenient if it did use the same URL (no need to pass a second URL to the automation harness), and it does make some sense as these messages are all about a specific test results collection, not the entirety of the job.

My first thoughts were to update the current "results" url to also look for `error` or `status: "TEST_START"` and update appropriately.